splat-radio.sh

```diff
@@ -36,8 +46,78 @@ if [ ! -x `which optipng` ]; then
 	exit 1
 fi
 
-# no more variables to edit below here
 
+function helptext {
+cat <<EOF
+
+Usage: $0 -c CONFIGFILE [-b]
+
+-c CONFIGFILE
+        The file with your configuration, see README.md
+        
```

Comment:
the ** -b ** option is used to render images with white background instead of topography hillshading.
This allows calling the script by create_overlay.sh because white background can easily converted to transparency.

**-r** is used to quickly change between a large scale image (splat) and a detailed image (splat-hd). So you can render images with either tool without changing the configfile.

**-R** is just a convenience so i can adjust rendering radius (and therefore time) for debugging

**-h** and **-c** are to make it easier with getopts and get consistency